src/test_register.c

```diff
@@ -123,6 +123,7 @@ int UT_register_tests(void)
         /* TODO: Have a dsHost Register function and call it always */
         registerFailed |= test_l1_dsHost_register();
         registerFailed |= test_l2_dsHost_register();
+        registerFailed |= test_l3_dsHost_register();
     }
     if (gDSModule & dsAudioPort)
     {
```

Comment:
remove this 
/* Register UT Functions */
int UT_register_APIDEF_l3_tests( void )
{
    int registerFailed=0;
    if(gDSModule & dsAudioPort)
        registerFailed |= test_l3_dsAudio_register();
    if(gDSModule & dsVideoPort)
        registerFailed |= test_l3_dsVideoPort_register();
        
    if(gDSModule & dsHost)
        registerFailed |= test_register_dsHost_hal_l3_tests();
    }
    return registerFailed;
}